pkg/userutils/set_uid_gid_1_15.go

```diff
@@ -0,0 +1,34 @@
+// +build linux
```

Comment:
`set_uid_gid_1_15.go ` -> `set_uid_gid_lower_go_ver.go ' or something?

Reply:
It follows the file naming convention like:
https://github.com/aws/aws-sdk-go/blob/master/aws/url.go
https://github.com/aws/aws-sdk-go/blob/master/aws/url_1_7.go

There is another style in golang code. We could change to it if preferred.
https://github.com/golang/go/blob/master/src/sort/slice_go14.go
https://github.com/golang/go/blob/master/src/sort/slice_go18.go
https://github.com/golang/go/blob/master/src/sort/slice_go113.go